src/Sulu/Component/Rest/ListBuilder/Expression/Doctrine/DoctrineNotInExpression.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Rest\ListBuilder\Expression\Doctrine;
+
+use Doctrine\ORM\QueryBuilder;
+
+/**
+ * Represents a NOT IN expression for doctrine - needs a field and an array of values.
+ */
+class DoctrineNotInExpression extends DoctrineInExpression
```

Comment:
Wouldn't it be nicer to have a simple `DoctrineNotExpression`, which can also be used with other statements than `IN`?

Reply:
@done in the ListBuilder we still need to have a `notIn` function